beacon-chain/rpc/validator/exit.go

```diff
@@ -15,7 +14,7 @@ import (
 )
 
 // ProposeExit proposes an exit for a validator.
-func (vs *Server) ProposeExit(ctx context.Context, req *ethpb.SignedVoluntaryExit) (*ptypes.Empty, error) {
+func (vs *Server) ProposeExit(ctx context.Context, req *ethpb.SignedVoluntaryExit) (*ethpb.ProposeExitResponse, error) {
```

Comment:
Note. I had to update this to comply with latest ethapi commit. #7125 was going to address this